planex/cmd/pin.py

```diff
@@ -147,6 +146,10 @@ def parse_args_or_exit(argv=None):
     parser.add_argument("--patchqueue", default="master",
                         help="Value for the patchqueue field of the pin file. "
                              "Defaults to master. ")
+    parser.add_argument("--o", default=None,
```

Comment:
`"-o", "--output"

Reply:
Yes, I should have done it from the beginning, sorry. Fixed